python/fledge/services/core/api/auth.py

```diff
@@ -734,15 +727,14 @@ async def _unblock_user(user_id, storage_client):
 
     from fledge.common.storage_client.payload_builder import PayloadBuilder
 
-    payload = PayloadBuilder().SELECT("id", "uname", "role_id", "block_until", "failed_attempts").WHERE(
+    payload = PayloadBuilder().SELECT("id").WHERE(
         ['id', '=', user_id]).payload()
     old_result = await storage_client.query_tbl_with_payload('users', payload)
     if len(old_result['rows']) == 0:
         raise User.DoesNotExist('User does not exist')
-    current_datetime = str(datetime.datetime.now())
 
     # Clear the failed_attempts so that maximum allowed attempts can be used correctly
-    payload = PayloadBuilder().SET(block_until=current_datetime, failed_attempts=0).WHERE(['id', '=', user_id]).payload()
+    payload = PayloadBuilder().SET(block_until="", failed_attempts=0).WHERE(['id', '=', user_id]).payload()
```

Comment:
This will break in PostgreSQL.

Reply:
> This will break in PostgreSQL.

We must fix this

Reply:
To insert null value to the database from Python, just use  'None' as it corresponds to NULL type.